package.json

```diff
@@ -26,5 +55,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "husky": "^4.2.5",
+    "lint-staged": "^10.2.11",
+    "prettier": "2.0.5"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{json,txt,ts,tsx,jsx,html,js,css,md}": ["prettier --write",
+    "eslint --fix",
+    "git add",
+    "jest --bail --findRelatedTests"]
```

Comment:
Are we using Jest or Tape? Why is there a `git add` in the middle of this.

Reply:
Removing....